src/transform/FooterLegal.css

```diff
@@ -0,0 +1,38 @@
+
+}
+
+html[dir="ltr"] .pagelib_footer_legal_contents {
```

Comment:
I don't mind if you want to use quotes but can we stick with single quotes? We currently use single quotes in all the JavaScript.